### PR TITLE
new icon: unrealengine (original, original-wordmark, plain, plain-wordmark) 

### PR DESCRIPTION
**Pull request by Snailedlt**
# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date

unreal engine branding guidelines -> https://www.unrealengine.com/en-US/branding
unreal engine logo download -> https://epicgames.box.com/s/c2m8idcyejqvg5mjf4e2q73b7jbaghft

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -4446,7 +4446,7 @@
                 "original-wordmark"
             ]
         },
-        "color": "#212b35",
+        "color": "#000000",
```
Comment (Panquesito7) [inline devicon.json:4449]:
```suggestion
        "color": "#212b35",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Let's keep the color versions, remove the white versions, and rename the color versions to `original` and `original-wordmark`. Let us know if you need any help! 🙂

File: devicon.json (hunk 1 above)
Comment (Snailedlt) [reply devicon.json:4449]: Will do, but I'll keep `#000000` color, since that's the original color

File: devicon.json [hunk 2]
```diff
@@ -4465,6 +4465,26 @@
...
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
+            ]
+        },
+        "color": "#000000",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:4486]:
```suggestion
      "aliases": [
          {
              "base": "original",
              "alias": "plain"
          },
          {
              "base": "original-wordmark",
              "alias": "plain-wordmark"
          }
      ]
```

File: devicon.json [hunk 3]
```diff
@@ -4465,6 +4465,26 @@
...
+            "game"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
```
Comment (Panquesito7) [inline devicon.json:4482]:
```suggestion
                "plain",
                "plain-wordmark"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: devicon.json [hunk 4]
```diff
@@ -4465,6 +4465,35 @@
...
+      "aliases": [
+          {
+              "base": "original",
+              "alias": "plain"
+          },
+          {
+              "base": "original-wordmark",
+              "alias": "plain-wordmark"
+          }
+      ]
```
Comment (Panquesito7) [inline devicon.json:4495]:
```suggestion
        "color": "#000000",
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            },
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
            }
        ]
```

**Comment by Thomas-Boi** [comment]
For now, we are running into refusing connection issue. I'll retry the script again tomorrow. Meanwhile, I'll also double check and see if a permanent fix for this is possible

**Comment by Thomas-Boi** [comment]
Due to the bugs related to `peek-bot` and `upload-artifact`, this PR will need longer than usual to be merged. Hopefully, it'll be merged within the next 2 weeks. Until then, thank you for your patience and understanding.

File: devicon.json [hunk 5]
```diff
@@ -4475,6 +4475,35 @@
...
+            "game"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
```
Comment (Thomas-Boi) [inline devicon.json:4492]:
```suggestion
                "original",
                "original-wordmark"
```
The `aliases` system is a bit confusing. Here, you need to put the actual SVG files that are used as the `font` versions. The `aliases` attributes below would then convert the names to its aliases when making the classes.

**Review by Thomas-Boi — APPROVED** [review]
@SnailedIt good work on the icon! It looks very clean and well done. The icon is ready to be merged once you decide which you want the icon to be. I'm leaning towards the `#000000` since that's what the original SVG version has. However, the white version as seen in their website is fine too. You can make the choice and let us know.

**Review by Panquesito7 — APPROVED** [review]
Amazing work! 🚀 Thanks so much for your contribution! 😄👍

**Comment by Thomas-Boi** [comment]
@Snailedlt seems like you are ok with the color. I'll just merge the PR in then.

**Comment by Snailedlt** [comment]
> @Snailedlt seems like you are ok with the color. I'll just merge the PR in then. 

Fine by me :)

**Comment by Snailedlt** [comment]
> @SnailedIt good work on the icon! It looks very clean and well done. The icon is ready to be merged once you decide which you want the icon to be. I'm leaning towards the `#000000` since that's what the original SVG version has. However, the white version as seen in their website is fine too. You can make the choice and let us know.

I think this shows that it would be a good feature to have more color options for logos. 

Maybe also more logo versions? Like vertical and horizontal watermark, instead of just watermark :)
Alternatively they could be called watermark and watermark-alternative-01, watermark-alternative-02, and so on (if there are multiple).

**Pull request merged**